PROJECT_CHARTER.md

```diff
@@ -0,0 +1,72 @@
+# WebdriverIO Charter
+
+## Section 0: Guiding Principles (optional)
+
+The WebdriverIO project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1: Scope
+
+The project tries to provide users a variety of building blocks to create a highly customizable, maintainable and scaleable framework to build and run end-to-end (e2e) tests for web and mobile applications. While it is focused to run these tests based on the [WebDriver](https://w3c.github.io/webdriver/) standard it leverages also other interfaces to provide as much control of the automated user agent as possible.
+
+With that the project offers not only the ability to automate user agents using open standards, it also enforces a set of common good practices that help to avoid pitfalls as well as the ability to connect predefined solutions to problem spaces that were already solved by the WebdriverIO community.
+
+### 1.1: In-scope
+
+- automation of modern web browser (e.g. Chrome, Firefox, Safari, Edge)
+- automation of mobile devices (web as well as native applications)
+- testing applications from end-to-end at scale (including parallisation and reporting)
+- integration into 3rd party services and vendors (e.g. [Sauce Labs](https://saucelabs.com/) or [Applitools](https://applitools.com/))
+- support of functional as well as non function testing aspects (e.g. visual regression or frontend performance testing)
```

Comment:
- support of functional as well as non functional testing aspects

Reply:
why removing the examples here?